### PR TITLE
Feathers API - api vs admin

### PR DESCRIPTION
**Pull request by djhi**
- [x] Separated the public api and the admin api
- [x] Fixed migrations to initialize the db
- [x] Fixed parts of the api e2e tests

### CODE REVIEW

File: e2e/api/public/apiOrders.js [hunk 1]
```diff
@@ -1,126 +1,128 @@
...
-                url: '/api/orders',
-            });
-            assert.equal(statusCode, 405);
-        });
-    });
-    after(function* removeFixtures() {
-        yield fixtureLoader.removeAllFixtures();
-    });
-});
+// import { assert } from 'chai';
```
Comment (fzaninotto) [inline e2e/api/public/apiOrders.js]: commenting out code is evil

File: e2e/api/public/apiOrdersId.js [hunk 2]
```diff
@@ -1,124 +1,126 @@
...
-            assert.equal(statusCode, 200, JSON.stringify(body));
-            userOrders = yield orderQueries.selectByUserId(user1.id);
-            assert.equal(userOrders.length, 1);
-        });
-    });
-    after(function* removeFixtures() {
-        yield fixtureLoader.removeAllFixtures();
-    });
-});
+// import { assert } from 'chai';
```
Comment (fzaninotto) [inline e2e/api/public/apiOrdersId.js]: pure evil. You're going Volskwagen on this.

File: e2e/api/public/apiOrders.js (hunk 1 above)
Comment (djhi) [inline e2e/api/public/apiOrders.js]: I know but I didnt want to loose it. I haven't worked on orders yet

File: e2e/api/public/apiOrdersId.js (hunk 2 above)
Comment (djhi) [inline e2e/api/public/apiOrdersId.js]: Right ^^ but for the same reason: I'm going to work on orders soon

**Pull request merged**